docs/migration/faqs.md

```diff
@@ -0,0 +1,203 @@
+</details>
+
+
+
+
+
+### Question 13: What support services will be available during and after the migration to address any issues or concerns?
+<details>
+  
+<summary> <b>Answer</b>: </summary>
+ The research facilitation team is committed to providing personalized assistance and support services during and after the migration:
+
+  - Personal Assistance:
+    - The research facilitation team is dedicated to offering personal assistance to each user. Whether you need help with data migration, code adjustments, or understanding the new environment, our team is here to provide tailored support.
+
+  - Issue Resolution:
+    - Any issues or concerns that arise during or after the migration will be promptly addressed by the research facilitation team. We aim to ensure a smooth transition for all users and are ready to tackle any challenges that may arise.
+
+  - Ongoing Support:
+    - Support services will continue to be available after the migration to address ongoing needs, answer questions, and assist with any further optimizations or adjustments required for your research projects.
+
+  Your success is our priority, and the research facilitation team is here to guide you through the migration process and beyond. If you encounter any issues or have specific concerns, please reach out to the team for personalized assistance.
+</details>
+
+
+
+
+
+### Question 14: Can I test my applications or simulations on Wulver before the migration to ensure compatibility?
+<details>
+  
+<summary> <b>Answer</b>: </summary>
+ Yes, absolutely! We encourage users to proactively test their applications or simulations on Wulver before the migration to ensure compatibility and identify any potential issues. This testing phase allows you to familiarize yourself with the new environment and address any concerns in advance.
+
+  If you encounter challenges or have questions during the testing process, please don't hesitate to reach out to us. Our team is here to provide guidance, answer queries, and assist you in ensuring a smooth transition for your research activities on Wulver. Your proactive testing will contribute to a successful migration experience.
+</details>
+
+
+
+
+
+### Question 15: How long will the `/oldhome` directory on Lochness be accessible after the migration, and what actions should I take regarding data stored there?
+<details>
+  
+<summary> <b>Answer</b>: </summary>
+ The `/oldhome` directory on Lochness will be accessible for 6 months after the migration is complete. During this period, users are advised to review and move their data to other locations on Wulver or archive it as needed. This timeframe provides a reasonable window for users to organize and transfer their data while ensuring a smooth transition.
```

Comment:
Should mention that /oldhome is read-only on Wulver. Also reiterate that /oldhome will not be accessible from Lochness once migration begins

Reply:
Added read only to the filesystem question. Since lochness itself is unavailable to user once they migrate no need to mention /oldhome not accessible.